src/goto-programs/goto_convert.cpp

```diff
@@ -1258,6 +1258,7 @@ void goto_convertt::convert_switch(
 
   goto_programt tmp_cases;
 
+  size_t case_number=1;
```

Comment:
As I may be lacking context: what sort of pretty printing is this about? Is it different from what dump-c and expr2{c,java} are doing at the moment?